### PR TITLE
Add Internal Bam SubreadSet and AlignmentSet

### PR DESCRIPTION
**Pull request by mdsmith**

### CODE REVIEW

File: data/AlignmentSet/m54075_160703_015941.alignmentset.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<AlignmentSet CreatedAt="2016-07-19T13:46:26" MetaType="PacBio.DataSet.AlignmentSet" Name="pacbio_dataset_alignmentset-160719_204626431" Tags="" TimeStampedName="pacbio_dataset_alignmentset-160719_204626431" UniqueId="5330b6dc-1c10-f996-5dbe-98a99634ecd5" Version="3.0.1" xmlns="http://pacificbiosciences.com/PacBioDatasets.xsd" xmlns:pbbase="http://pacificbiosciences.com/PacBioBaseDataModel.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://pacificbiosciences.com/PacBioDataModel.xsd">
```
Comment (natechols) [inline data/AlignmentSet/m54075_160703_015941.alignmentset.xml]: Can you make the name the same as the ID that will retrieve this?

File: data/SubreadSet/m54075_160703_015941.subreadset.xml [hunk 2]
```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<pbds:SubreadSet UniqueId="5243fae9-cb9d-4c44-8b4d-0ac1b5cb1b30" MetaType="PacBio.DataSet.SubreadSet" TimeStampedName="SubreadSetCollection_160703_073656974" CreatedAt="2016-07-03T07:36:56Z" Name="SMS_FleaVer_54075_15kEcoli_2kLambda_2" Tags="subreadset" Version="3.0.1" xmlns="http://pacificbiosciences.com/PacBioBaseDataModel.xsd" xmlns:pbdm="http://pacificbiosciences.com/PacBioDataModel.xsd" xmlns:pbds="http://pacificbiosciences.com/PacBioDatasets.xsd" xmlns:pbmeta="http://pacificbiosciences.com/PacBioCollectionMetadata.xsd" xmlns:pbsample="http://pacificbiosciences.com/PacBioSampleInfo.xsd" xmlns:pbrk="http://pacificbiosciences.com/PacBioReagentKit.xsd" xmlns:pbpn="http://pacificbiosciences.com/PacBioPartNumbers.xsd" xmlns:pbbase="http://pacificbiosciences.com/PacBioBaseDataModel.xsd">
```
Comment (natechols) [inline data/SubreadSet/m54075_160703_015941.subreadset.xml]: Again, the name should probably be the same as the ID in files.json

**Comment by mdsmith** [comment]
Good to know. Have you defined a process for adding, maintaining and updating these files? If not we definitely should.

**Comment by natechols** [comment]
No, I've been the only person adding them so far.  Probably time to update the README.

**Comment by natechols** [comment]
LGTM.

**Pull request merged**